modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSetAdmin.macro

```diff
@@ -1,5 +1,15 @@
 definition {
 
+	macro cancelDeleteDataSetView {
+		Click(
+			key_name = ${viewName},
+			locator1 = "FrontendDataSetAdmin#DATA_SET_VIEW_ENTRY_KEBAB_MENU");
+
+		ClickNoError(locator1 = "ObjectPortlet#DELETE_ENTRY_BUTTON");
+
+		Button.clickCancel();
+	}
```

Comment:
Hey @andreluis2022 this function is only used once, so you don't need to create this macro, you can put these steps directly into the test.

Reply:
ohh,Ok @mateusralv